src/Transport/GrpcTransport.php

```diff
@@ -144,7 +157,10 @@ public static function build(string $apiEndpoint, array $config = [])
             );
         }
         try {
-            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors']);
+            if (isset($config['logger']) && $config['logger'] === false) {
+                $config['logger'] = null;
+            }
+            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors'], $config['logger'] ?? null);
```

Comment:
Add `'logger' => null` to the `$config +=` array above, and then this can be simpler.
```suggestion
            if ($config['logger'] === false) {
                $config['logger'] = null;
            }
            return new GrpcTransport($host, $stubOpts, $channel, $config['interceptors'], $config['logger']);
```